libmdnsd/CMakeLists.txt

```
@@ -63,16 +68,12 @@ if (MDNSD_BUILD_FUZZING_CORPUS)
 endif()
 
 if(MDNSD_COMPILE_AS_CXX)
-  set_source_files_properties(${LIBRARY_FILES} PROPERTIES LANGUAGE CXX)
+    set_source_files_properties(${LIBRARY_FILES} PROPERTIES LANGUAGE CXX)
 endif()
 
-if(MDNSD_BUILD_OSS_FUZZ)
-    add_library(libmdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
-else()
-    add_library(libmdnsd SHARED ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
-endif()
+add_library(mdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
```

```suggestion
add_library(libmdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
set_property(TARGET libmdnsd PROPERTY OUTPUT_NAME "mdnsd")
```
See https://cmake.org/cmake/help/latest/prop_tgt/OUTPUT_NAME.html